standard/ERCs/erc-6900.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 6900
-title: Modular Smart Contract Accounts and Plugins
+title: Modular Smart Contract Accounts and Modules
 description: Interfaces for composable contract accounts optionally supporting upgradability and introspection
 author: Adam Egyed (@adamegyed), Fangting Liu (@trinity-0111), Jay Paik (@jaypaik), Yoav Weiss (@yoavw)
-discussions-to: https://ethereum-magicians.org/t/eip-modular-smart-contract-accounts-and-plugins/13885
+discussions-to: https://ethereum-magicians.org/t/eip-modular-smart-contract-accounts-and-modules/13885
```

Comment:
haha this link should not be updated, seems like a side effect of the Ctrl-F replace